src/ec.rs

```diff
@@ -127,10 +127,60 @@ pub struct WriterBase<S> {
   s: S,
 }
 
+/*
+ * import math
+ * def lr_compute(rngShift, diff):
+ *     u = ((rngShift * diff) >> 1) + 4
+ *     bits = 15 - math.floor(math.log2(u))
+ *     u = u << bits
+ *     return ((u >> 8) - 128, bits)
+ *
+ * def stats():
+ *     for diff in range(0, 513):
+ *         avg = sum([lr_compute(r, diff)[1] for r in range(128, 256)])
+ *         yield(avg)
+ *
+ * print([x for x in (stats())])
+ */
+// Units of 1/128 of a bit
+const ENTROPY_LOOKUP: [u16; 513] = [
```

Comment:
This looks a bit like a subtle performance bug that has popped up before.
```suggestion
const ENTROPY_LOOKUP: &[u16; 513] = &[
```

Reply:
Wow I had no idea that this could lead to performance issues. I'm really curious if there's any Github issue on Rustc talking about this.